src/ctrlc.rs

```diff
@@ -0,0 +1,73 @@
+use console::Term;
+use once_cell::sync::Lazy;
+use signal_hook::{
+    consts::SIGINT,
+    iterator::{Handle, Signals},
+};
+use std::{
+    io::Error,
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Mutex, RwLock,
+    },
+    thread,
+};
+
+static INIT: AtomicBool = AtomicBool::new(false);
+static INIT_LOCK: Mutex<()> = Mutex::new(());
+
+static HANDLE: Lazy<RwLock<Option<Handle>>> = Lazy::new(|| RwLock::new(None));
```

Comment:
```suggestion
static HANDLE: Lazy<RwLock<Option<Handle>>> = Lazy::new(Default::default);
```

this does the same thing, but if we ever decide to use a different mutex type or something it makes refactoring easier